api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/processor/DetermineEntitlementMessageProcessor.java

```diff
@@ -82,8 +104,16 @@ private void handleDecision(Claim claim, PaymentCycle previousPaymentCycle, Paym
             }
         } else if (claim.getClaimStatus() == ACTIVE) {
             eligibilityDecisionHandler.handleIneligibleDecision(claim, previousPaymentCycle, currentPaymentCycle, decision);
+        } else if (claimHasBeenPendingExpiryForLongerThanTheMaximumDuration(claim, messageTimestamp)) {
+            claim.updateClaimStatus(EXPIRED);
```

Comment:
This needs an explicit `save(claim)`. Sadly I think this might need another component added as we don't seem to have the `claimRepository` injected in this class.

Reply:
Added explicit save.